to_multi_var

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+ROOT_DIRECTORY="$( dirname "$( readlink -f "${BASH_SOURCE[0]}" )" )"
+readonly ROOT_DIRECTORY
```

Comment:
This prevents the reassignment of the ROOT_DIRECTORY variable. It is recommended to make variables readonly where possible. See https://google.github.io/styleguide/shell.xml?showone=Read-only_Variables#Read-only_Variables